src/sourceItemWrapper.js

```diff
@@ -489,20 +489,92 @@ class SourceItemWrapper extends ItemWrapper {
         // offer to automatically link to zotero items
     }
 
-    getFromBibTeX() {
-        Services.prompt.alert(
-            window,
-            Wikicite.getString('wikicite.global.unsupported'),
-            Wikicite.getString('wikicite.bibtex.import-citations.unsupported')
```

Comment:
Please remove this message (and the corresponding export one) from the English locale.